webtau-core/src/main/java/com/twosigma/webtau/Ddjt.java

```diff
@@ -69,6 +71,10 @@ public static ContainMatcher containing(Object expected) {
         return new ContainMatcher(expected);
     }
 
+    public static <K, V> Map<K, V> createMap(Object... kvs) {
```

Comment:
Would it be better to take a vararg of pairs (or map entries)?  We could define a static helper so that a user would do:
```
createMap(
    entry("foo", "bar"),
    entry("hello", "world")
)
```
We can then get compile time type checks instead of throwing a class cast exception and the runtime `% 2` check also goes away.

Reply:
fair. Initially it was a private method to help with impl details. Entry could be to generic for a static import though. Java doesn't have the Kotlin context magic. And it is being for testing only.